src/main/java/banking/configuration/DBManager.java

```diff
@@ -0,0 +1,43 @@
+package banking.configuration;
+
+import org.sqlite.SQLiteDataSource;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class DBManager {
+
+    private final static String CREATE_TABLE = """
+            CREATE TABLE IF NOT EXISTS account
+            (id INTEGER PRIMARY KEY, 
+            number TEXT NOT NULL, 
+            pin TEXT NOT NULL, 
+            balance INTEGER DEFAULT 0)""";
+    private String dbName;
+    private String url;
+    private SQLiteDataSource dataSource = new SQLiteDataSource();
+
+    public DBManager(String dbName) {
+        this.dbName = dbName;
+        createDatabase();
+        createTable();
+    }
+
+    public void createDatabase() {
+        url = "jdbc:sqlite:" + dbName;
```

Comment:
The `dbName ` can have any file extension right now. Can you fix that?

Reply:
fixed